### PR TITLE
[Network] Bump to azure-mgmt-network 8.0.0

### LINKED ISSUE #11988: [Network] Fix actions required for az network private-link-service connection update and bump to 9.0.0
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by MyronFanQiu**
Fix #11988

[Network] Bump to azure-mgmt-network 8.0.0
[Network] az network private-link-service update/create: support --enable-proxy-protocol

**History Notes:**  
(Fill in the following template if multiple notes are needed, otherwise PR title will be used for history note.)

[Component Name 1] (BREAKING CHANGE:) (az command:) make some customer-facing change.  
[Component Name 2] (BREAKING CHANGE:) (az command:) make some customer-facing change.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -82,6 +82,7 @@ Release History
 **Network**
 
 * Fix #2092: az network dns record-set add/remove: add warning when record-set is not found. In the future, an extra argument will be supported to confirm this auto creation.
+* Bump azure-mgmt-network to 9.0.0
```
Comment (fengzhou-msft) [inline src/azure-cli/HISTORY.rst]: This could be removed now.

Comment (MyronFanQiu) [reply src/azure-cli/HISTORY.rst]: ok.

File: tools/automation/tests/main.py [hunk 2]
```diff
@@ -33,7 +33,7 @@ def run_tests(modules, parallel, run_live, tests):
     test_paths = tests or [p for _, _, p in modules]
 
     display('Drive test by nosetests')
-    runner = get_nose_runner(parallel=parallel, process_timeout=3600 if run_live else 600)
+    runner = get_nose_runner(parallel=parallel, process_timeout=3600 if run_live else 1200)
```
Comment (MyronFanQiu) [inline tools/automation/tests/main.py]: @fengzhou-msft do you know how to debug this script?

Comment (fengzhou-msft) [reply tools/automation/tests/main.py]: I haven't worked with nose runner, ping @haroldrandom

File: src/azure-cli/azure/cli/command_modules/network/_params.py [hunk 3]
```diff
@@ -760,6 +760,7 @@ def load_arguments(self, _):
         c.argument('private_endpoint_connections', nargs='+', help='Space-separated list of private endpoint connections.')
         c.argument('fqdns', nargs='+', help='Space-separated list of FQDNs.')
         c.argument('location', get_location_type(self.cli_ctx), validator=get_default_location_from_resource_group)
+        c.argument('enable_proxy_protocol', help='Enable proxy protocol for private link service.', arg_type=get_three_state_flag(), min_api='2019-09-01')
```
Comment (haroldrandom) [inline src/azure-cli/azure/cli/command_modules/network/_params.py]: Is it new feature?

Comment (MyronFanQiu) [reply src/azure-cli/azure/cli/command_modules/network/_params.py]: yes

**Pull request merged**